app/components/course-rollover.js

```diff
@@ -79,7 +79,7 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
     if (expandAdvancedOptions && skipOfferings) {
       data.skipOfferings = true;
     }
-    const host = this.get('host')?this.get('host'):'/';
+    const host = this.get('host') ? (this.get('host').replace(/\/+$/, '') + '/') : '/';
```

Comment:
This feels like it can be applied generally and belongs in the config service instead of dealing with it every time we ask for the host somewhere.